content/doc/book/installing/_setup-wizard.adoc

```diff
@@ -13,30 +13,29 @@ _run-jenkins-in-docker.adoc).
 [[setup-wizard]]
 == Post-installation setup wizard
 
-After downloading, installing and running Jenkins using one of the procedures
-above (except for installation with Jenkins Operator), the post-installation setup wizard begins.
+After downloading, installing, and running Jenkins using one of the procedures
+above (except when installing with the Jenkins Operator), the post-installation setup wizard begins.
 
-This setup wizard takes you through a few quick "one-off" steps to unlock
-Jenkins, customize it with plugins and create the first administrator user
+The setup wizard takes you through a few quick one-time steps to unlock Jenkins, customize it with plugins and create the first administrator user
```

Comment:
Since you've used Oxford comma elsewhere, should it be used here as well?  Optional
```suggestion
The setup wizard takes you through a few quick one-time steps to unlock Jenkins, customize it with plugins, and create the first administrator user
```